doc/introduction/interfaces/numpy.rst

```diff
@@ -53,8 +53,8 @@ returning NumPy arrays.
 
 It can now be used like any other Python/NumPy function:
 
->>> phi = np.array([0.5, 0.1])
->>> theta = 0.2
+>>> phi = np.array([0.5, 0.1], requires_grad=True)
+>>> theta = np.array(0.2, requires_grad=True)
```

Comment:
How come these were changed?

Reply:
@antalszava I think you are right it is not necessary as we don't use `qml.grad` after it.